src/es6-init.js

```diff
@@ -10,7 +10,7 @@ function findPackageJson(initScript) {
   // Walk up the parent directories until we find package.json. Make sure that
   // we're not actually stumbling upon a parent npm package
   let ret = path.join(initScript, 'package.json')
-  if (fs.statSyncNoException(ret) && !path.resolve(path.dirname(ret), '..').match(/[\\\/]node_modules$/i)) {
+  if (_fs2.default.statSyncNoException(ret) && (initScript.startsWith("/") || !_path2.default.resolve(_path2.default.dirname(ret), '..').match(/[\\\/]node_modules$/i))) {
```

Comment:
Were these built-in module name changes intentional? There aren't any `_fs2` or `_path2` identifiers in this file.